pkg/kubelet/server/server.go

```diff
@@ -214,14 +214,15 @@ type PodResourcesProviders struct {
 	Devices podresources.DevicesProvider
 	Cpus    podresources.CPUsProvider
 	Memory  podresources.MemoryProvider
+	DRA     podresources.DRAProvider
 }
 
 // ListenAndServePodResources initializes a gRPC server to serve the PodResources service
 func ListenAndServePodResources(socket string, providers PodResourcesProviders) {
 	server := grpc.NewServer()
 
 	podresourcesapiv1alpha1.RegisterPodResourcesListerServer(server, podresources.NewV1alpha1PodResourcesServer(providers.Pods, providers.Devices))
-	podresourcesapi.RegisterPodResourcesListerServer(server, podresources.NewV1PodResourcesServer(providers.Pods, providers.Devices, providers.Cpus, providers.Memory))
+	podresourcesapi.RegisterPodResourcesListerServer(server, podresources.NewV1PodResourcesServer(providers.Pods, providers.Devices, providers.Cpus, providers.Memory, providers.DRA))
```

Comment:
This will go away if we start passing the struct instead of each individual provider.